examples/sso-to-pingdirectory-from-pingone/README.md

```diff
@@ -0,0 +1,43 @@
+# Setting up SSO to PingDirectory from PingOne
+
+Reference: [Setting up SSO to PingDirectory from PingOne](https://docs.pingidentity.com/r/en-us/pingone/pd_ds_set_up_sso_pingdir_pingone)
+
+## Before you begin
+
+* A PingDirectory server configured for Terraform access. This server will host the admin console that is being configured for SSO.  For more information, see [Getting Started - PingDirectory](https://terraform.pingidentity.com/getting-started/pingdirectory/)
+* A PingOne account configured for Terraform access.  For more information, see [Getting Started - PingOne](https://terraform.pingidentity.com/getting-started/pingone/)
+
+## Setting Connection Details
+The following environment variables should be set prior to running the example.  For more information, see the registry documentation for each provider.
+
+| Variable name                                   | Schema Documentation                                                                                                                            | Required/Optional |
+|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| `PINGDIRECTORY_PROVIDER_USERNAME`               | [PingDirectory - username](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#username)                             | Required          |
+| `PINGDIRECTORY_PROVIDER_PASSWORD`               | [PingDirectory - password](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#password)                             | Required          |
+| `PINGDIRECTORY_PROVIDER_HTTPS_HOST`             | [PingDirectory - http_host](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#https_host)                          | Required          |
+| `PINGDIRECTORY_PROVIDER_INSECURE_TRUST_ALL_TLS` | [PingDirectory - insecure_trust_all_tls](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#insecure_trust_all_tls) | Optional          |
```

Comment:
I suppose since it is just an example it is fine to recommend this, but maybe we could mention that this is just for testing and you should use actual cert configuration (https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#ca_certificate_pem_files) in production

Reply:
Good point - I'll add that in to the next draft